src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp.Client/Program.Main.cs

```diff
@@ -0,0 +1,23 @@
+#if (IndividualLocalAuth)
+using BlazorWeb_CSharp.Client;
+using Microsoft.AspNetCore.Components.Authorization;
+#endif
+using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+
+namespace BlazorWeb_CSharp.Client;
+
+class Program
```

Comment:
Unlike the server template, this does not have an accessibility modifier. Neither does the `Main` method. I would have made both the class and `public` like we do for the server project, but this made the `@typeof(Program).Assembly` reference in Routes.razor ambiguous.

https://github.com/dotnet/aspnetcore/blob/d8acc61aa53bbc5ed18749677d2afa6bd38fcc31/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Routes.razor#L5

I could change that to be `@typeof(BlazorWeb_CSharp.Program).Assembly` instead and make this public. And then only qualify it when you've disabled top level statements.

Does anyone prefer that, or is everyone fine the way it is?